src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/LoadFromBytes.xaml.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+using Uno.Extensions;
+
+namespace Uno.UI.Samples.UITests.ImageTestsControl
+{
+	[SampleControlInfo("Image", "LoadFromBytes")]
+	public sealed partial class LoadFromBytes
+	{
+		public LoadFromBytes()
+		{
+			InitializeComponent();
+			Loaded += OnLoaded;
+		}
+
+		private async void OnLoaded(object sender, RoutedEventArgs e)
+		{
+			var assembly = typeof(LoadFromBytes).Assembly;
+
+			var query = from name in assembly.GetManifestResourceNames()
+						where name.EndsWith("mslug.png")
+						select assembly.GetManifestResourceStream(name);
+
+			using (var stream = query.First())
+			{
+				MyImage.Source = await CreateImageSource(stream);
+			}
+		}
+
+		private async Task<ImageSource> CreateImageSource(Stream stream)
```

Comment:
Since it's quite common to create an ImageSource from a stream, would it be OK to make this reusable somewhere? Like an utility helper method?

Reply:
This is the sort of helpers we generally place in [Uno.Core](https://github.com/unoplatform/Uno.Core). The problem here is mostly about how to find the proper asset name.

That could be an extension like `var s = assembly.GetResourceStream(n => n.EndsWith("mslug.png")`.

Reply:
That's OK! but I was referring to the method
```
private async Task<ImageSource> CreateImageSource(Stream stream)
```
I think it could be a good addition to centralize loading a `BitmapSource` from a `Stream`. But that could be in another PR, if you think it's OK.

Reply:
That'll be fine for now.